### PR TITLE
JAMES-2494 Remove Kafka

### PR DESCRIPTION
**Pull request by blackheaven**

### CODE REVIEW

File: mailbox/spring/src/test/resources/events.properties [hunk 1]
```diff
@@ -20,12 +20,7 @@
 #  DO NOT USE IT AS SUCH AND ADAPT IT TO YOUR NEEDS
 
 # The following properties needs to be set
-# if you want to use Kafka for inter node messaging for your event system.
-#
-# This needs to be done if you used <publisher>kafka</publisher> in event.xml
 
-kafka.ip=127.0.0.1
-kafka.port=9092
```
Comment (rouazana) [inline mailbox/spring/src/test/resources/events.properties]: it seems you can remove completely this file, no?

File: server/app/src/main/resources/events.properties [hunk 2]
```diff
@@ -20,12 +20,7 @@
 #  DO NOT USE IT AS SUCH AND ADAPT IT TO YOUR NEEDS
 
 # The following properties needs to be set
-# if you want to use Kafka for inter node messaging for your event system.
-#
-# This needs to be done if you used <publisher>kafka</publisher> in event.xml
 
-kafka.ip=127.0.0.1
-kafka.port=9092
```
Comment (rouazana) [inline server/app/src/main/resources/events.properties]: idem?

File: src/site/xdoc/server/config-events.xml [hunk 3]
```diff
@@ -115,8 +115,6 @@
             <p>There are two types of event serialization systems :
```
Comment (rouazana) [inline src/site/xdoc/server/config-events.xml]: so there is no more 2 types :)

**Comment by chibenwa** [comment]
test this please (build output lost)

**Comment by chibenwa** [comment]
```
[e0defb5daa6d2877710ff93a9eb33a3d121356a7] Results :
[e0defb5daa6d2877710ff93a9eb33a3d121356a7] 
[e0defb5daa6d2877710ff93a9eb33a3d121356a7] Tests in error: 
[e0defb5daa6d2877710ff93a9eb33a3d121356a7]   SpringMailboxTest.beforeClass:35 » BeanInitialization Could not load propertie...
[e0defb5daa6d2877710ff93a9eb33a3d121356a7] 
[e0defb5daa6d2877710ff93a9eb33a3d121356a7] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0
```

**Comment by chibenwa** [comment]
```
[042795c2604edb6d9eae146d420cf0526b46e95f] org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'mailprocessor': Invocation of init method failed; nested exception is org.apache.commons.configuration.ConfigurationException: Unable to init mailet ToSenderFolder
[042795c2604edb6d9eae146d420cf0526b46e95f] 	at org.apache.james.app.spring.JamesAppSpringMainTest.testServer(JamesAppSpringMainTest.java:27)
[042795c2604edb6d9eae146d420cf0526b46e95f] Caused by: org.apache.commons.configuration.ConfigurationException: Unable to init mailet ToSenderFolder
[042795c2604edb6d9eae146d420cf0526b46e95f] 	at org.apache.james.app.spring.JamesAppSpringMainTest.testServer(JamesAppSpringMainTest.java:27)
[042795c2604edb6d9eae146d420cf0526b46e95f] Caused by: org.apache.mailet.MailetException: Could not load mailet (ToSenderFolder)
[042795c2604edb6d9eae146d420cf0526b46e95f] 	at org.apache.james.app.spring.JamesAppSpringMainTest.testServer(JamesAppSpringMainTest.java:27)
[042795c2604edb6d9eae146d420cf0526b46e95f] Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'org.apache.james.transport.mailets.ToSenderFolder': Unsatisfied dependency expressed through constructor argument with index 1 of type [org.apache.james.mailbox.MailboxManager]: : Error creating bean with name 'jpa-mailboxmanager' defined in class path resource [META-INF/spring/mailbox-jpa.xml]: Cannot resolve reference to bean 'delegating-listener' while setting constructor argument; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No bean named 'delegating-listener' is defined; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'jpa-mailboxmanager' defined in class path resource [META-INF/spring/mailbox-jpa.xml]: Cannot resolve reference to bean 'delegating-listener' while setting constructor argument; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No bean named 'delegating-listener' is defined
[042795c2604edb6d9eae146d420cf0526b46e95f] 	at org.apache.james.app.spring.JamesAppSpringMainTest.testServer(JamesAppSpringMainTest.java:27)
[042795c2604edb6d9eae146d420cf0526b46e95f] Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'jpa-mailboxmanager' defined in class path resource [META-INF/spring/mailbox-jpa.xml]: Cannot resolve reference to bean 'delegating-listener' while setting constructor argument; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No bean named 'delegating-listener' is defined
[042795c2604edb6d9eae146d420cf0526b46e95f] 	at org.apache.james.app.spring.JamesAppSpringMainTest.testServer(JamesAppSpringMainTest.java:27)
[042795c2604edb6d9eae146d420cf0526b46e95f] Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No bean named 'delegating-listener' is defined
[042795c2604edb6d9eae146d420cf0526b46e95f] 	at org.apache.james.app.spring.JamesAppSpringMainTest.testServer(JamesAppSpringMainTest.java:27)
[042795c2604edb6d9eae146d420cf0526b46e95f] 
[042795c2604edb6d9eae146d420cf0526b46e95f] 
[042795c2604edb6d9eae146d420cf0526b46e95f] Results :
[042795c2604edb6d9eae146d420cf0526b46e95f] 
[042795c2604edb6d9eae146d420cf0526b46e95f] Failed tests: 
[042795c2604edb6d9eae146d420cf0526b46e95f]   JamesSpringContextTest.springShouldLoadAndAddOnlyOneQuotaUpdaterListener Multiple Failures (2 failures)
[042795c2604edb6d9eae146d420cf0526b46e95f] 	Error creating bean with name 'mailprocessor': Invocation of init method failed; nested exception is org.apache.commons.configuration.ConfigurationException: Unable to init mailet ToSenderFolder
[042795c2604edb6d9eae146d420cf0526b46e95f] 	<no message> in java.lang.NullPointerException
[042795c2604edb6d9eae146d420cf0526b46e95f] Tests in error: 
[042795c2604edb6d9eae146d420cf0526b46e95f]   JamesAppSpringMainTest.testServer:27 » BeanCreation Error creating bean with n...
```

File: dockerfiles/run/spring/destination/conf/events.properties [hunk 4]
```diff
@@ -20,12 +20,7 @@
 #  DO NOT USE IT AS SUCH AND ADAPT IT TO YOUR NEEDS
```
Comment (rouazana) [inline dockerfiles/run/spring/destination/conf/events.properties]: this file should be removed too

**Comment by chibenwa** [comment]
merged

**Pull request closed without merging**